vsts-api-patcher/src/main.rs

```diff
@@ -481,6 +483,260 @@ impl Patcher {
         }
     }
 
+    fn patch_operation_status_in_releases_query_parameters(
+        &mut self,
+        key: &[&str],
+        _value: &JsonValue,
+    ) -> Option<JsonValue> {
+        // Only applies to pipelines specs
+        if !self.spec_path.ends_with("release.json") {
+            return None;
+        }
+        match key {
+            ["definitions", "DeploymentQueryParameters", "properties", "operationStatus"] => {
```

Comment:
Looks like this is pretty much the same patch (just with a different description) being applied to two places.

I think we should avoid code duplication and do both of the patches in a single function (e.g. `patch_release_operation_status(...)`), and have that call a single function to return the json with a parameter for the description....

```rust
fn patch_release_operation_status(...) {
    ...
    match key {
            ["definitions", "DeploymentQueryParameters", "properties", "operationStatus"] => {
                Some(patched_operation_status("Query deployment based on deployment operation status.")
            },
            ["definitions", "DeploymentAttempt", "properties", "operationStatus"] => {
                Some(patched_operation_status("Deployment operation status."))
            }
            ...
        }   
```

Reply:
Updated in the latest commit